com.unity.shadergraph/Editor/Data/Implementation/NodeUtils.cs

```diff
@@ -994,5 +995,91 @@ public static string FloatToShaderValueShaderLabSafe(float value)
             // For single point precision, reserve 54 spaces (e-45 min + ~9 digit precision). See floating-point-numeric-types (Microsoft docs).
             return value.ToString("0.######################################################", CultureInfo.InvariantCulture);
         }
+
```

Comment:
This is simply moving this code to a shared utility (out of PreviewManager.cs).  No changes were made to the code.